# ArgumentOutOfRangeError

- extends: [ArgumentError](./doc/api/python/exceptions/argumenterror.md)

**ArgumentOutOfRangeError is raised when the value of an argument is outside the allowable range of values as defined by the invoked method.**

## Constructor<!-- {docsify-ignore} -->
- [message](#message)
- [stacktrace](#stacktrace)


### message
- type: str

Message of the exception.


### stacktrace
- type: str

Stack of the exception which got raised inside the exception.